### ISSUE TITLE
FilePond itemInsertLocation not affecting addFile(s) 

### ISSUE THREAD

**Issue by danielxu7** [pqina/filepond#292]
### Summary

Hello, currently I have itemInsertLocation set to "after" to reduce performance issues. I have images from another part of the page and when calling pond.addFile(s), the images are added in front as if itemInsertLocation was set to "before".

### How to reproduce

```
FilePond.setOptions({ itemInsertLocation: "after" });
```

In another part of my javascript code, I have:

```
myFilePond.addfile(someDataURL);
```

### Expected behaviour

Adding more files should insert each image at the end of my list of images.

Is it possible for adding other files without clicking on the FilePond area itself and inserting it at the end of the list?

**Comment by rikschennink** [comment]
By default it should indeed add the file to the end of the list, it seems like I haven't updated the logic behind the `addFile` method.

Until this is fixed you can manually supply an `index` indicating where to add the file to the `addFile` method.

**Comment by danielxu7** [comment]
Thank you, your solution worked!

**Comment by rikschennink** [comment]
Fixed in version `4.4.0`
